3scale/porta
# Ditching yarn

What this PR does / why we need it:
Since we can't productize Yarn because Ocaml, we decided to commit our webpacker packs in our codebase as temporary solution. This PR fixes it removing Yarn and using npm instead.

Verification steps

    Circle CI should be green ;)
    You could try it locally and/or deploy to OS and confirm the webpack assets are being served.
